docs/_helpers/bandpass_page.py

```diff
@@ -26,7 +26,11 @@ def bandpass_table(setname):
     allrefs = []
 
     for m in bandpass_meta:
-        if m['filterset'] != setname:
+        if (
+                m['filterset'] != setname or
+                # special case of ztf position-dependent bandpass
```

Comment:
Is there a way to write this to be more generic rather than need to keep a list of all the filter systems with positional-dependent bandpasses?